packages/grid/_modules_/grid/models/params/gridCellParams.ts

```diff
@@ -1,12 +1,15 @@
 import { GridCellMode, GridCellValue } from '../gridCell';
-import { GridRowId, GridRowModel } from '../gridRows';
+import { GridRowDefaultData, GridRowId, GridRowModel } from '../gridRows';
 import type { GridStateColDef } from '../colDef';
 import { GridEditCellProps } from '../gridEditRowModel';
 
 /**
  * Object passed as parameter in the column [[GridColDef]] cell renderer.
  */
-export interface GridCellParams {
+export interface GridCellParams<
+  T = GridCellValue,
+  RowType extends GridRowDefaultData = GridRowDefaultData,
```

Comment:
I would reuse `GridRowModel` as much as possible.

```suggestion
  RowType extends GridRowModel = GridRowModel,
```